charts/posthog/templates/beat-hpa.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.beat.hpa.enabled -}}
```

Comment:
Why does celery beat need autoscaling in the first place?

Reply:
Will remove. I wasn't sure if this could run into scaling problems at one point if we have small pods, but seems like multiple instances aren't necessarily safe to run which takes priority.